web-client/public/js/setup-handlers.js

```diff
@@ -110,38 +108,38 @@ export const setupHandlers = grnState => {
                     }
                 }
             };
-            visit(svg);
+            visit(node);
             return tree;
         };
 
         const explicitlySetStyle = element => {
             const cSSStyleDeclarationComputed = window.getComputedStyle(element);
-            let i;
-            let len;
-            let key;
-            let value;
             let computedStyleStr = "";
 
-            for (i = 0, len = cSSStyleDeclarationComputed.length; i < len; i++) {
-                key = cSSStyleDeclarationComputed[i];
-                value = cSSStyleDeclarationComputed.getPropertyValue(key);
+            for (let i = 0; i < cSSStyleDeclarationComputed.length; i++) {
+                const key = cSSStyleDeclarationComputed[i];
+                const value = cSSStyleDeclarationComputed.getPropertyValue(key);
                 if (value !== emptySvgDeclarationComputed.getPropertyValue(key)) {
                     // Don't set computed style of width and height. Makes SVG elmements disappear.
                     if ((key !== "height") && (key !== "width")) {
-                        computedStyleStr += key + ":" + value + ";";
+                        computedStyleStr += `${key}:${value};`;
```

Comment:
Great change! Yes, in the age of template strings I generally change any concatenations `+` into template/formatted strings 💯